runtime.md

```diff
@@ -13,6 +13,13 @@ The state of a container MUST include, at least, the following properties:
 * **`id`**: (string) is the container's ID.
 This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
+* **`status`**: (string) is the runtime state of the container.
+The value MAY be one of:
+  * `created` : the container has been created but the user-specified process has not yet been executed
+  * `running` : the container has been created and the user-specified process is running
+  * `stopped` : the container has been created and the user-specified process has been executed but is no longer running
```

Comment:
If we land a `status` property (I'm still [opposed][1]), I think we want to use `dead` here to mirror proc(5) and ps(1).  More details on why I don't like “stopped” in #465.

[1]: https://github.com/opencontainers/runtime-spec/pull/462#issuecomment-222249696